### PR TITLE
V1 blog post

### PR DESCRIPTION
**Pull request by johno**
Here's what I've spiked out for the v1 announcement blog post. Would love to hear any thoughts, feedback, or anything that might be missing.

### CODE REVIEW

File: docs/blog/v1.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+[MDX Blog](/blog)
+
+# MDX goes stable
+
+It’s been a year and a half since the first MDX commit and a year since MDX was first announced at
+ZEIT Day.  MDX is a radical paradigm shift in how to write immersive content using components.  It’s
+an open, [authorable format](https://johno.com/authorable-format) that makes it _fun_ to write again.
+
+Over the last year we’ve been working on numerous bug fixes, new features, better parsing, and integration
```
Comment (wooorm) [inline docs/blog/v1.md]:
```suggestion
Since announcing MDX we’ve been working on numerous bug fixes, new features, better parsing, and integration
```

(reference back to earlier paragraph)

File: docs/blog/v1.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+## 🎉 What’s new?
+
+Here’s a rough breakdown on what we’ve been working on:
+
+### Parsing
+
+We’ve put a lot of work into improving the parsing of MDX documents.  This not only includes better
+handling of import and export blocks, but being more flexible with JSX blocks.  We won’t get into the
+nitty gritty here, but rest assured we’ve added many more tests and are handling all the edge cases
+we’ve currently encountered (please open an issue if you find a new parse issue 😸).
```
Comment (wooorm) [inline docs/blog/v1.md]: Attempt at a rewrite, not necessarily better!

```txt
Parsing of MDX documents is significantly improved.
We won’t get into the nitty gritty here, but we’ve seen how MDX is used in real-world projects, integrated the edge cases we came across into our test suite, and now handle JSX, imports, and exports much more intuitively.
Please open an issue if you find a case we haven’t covered! 😸
```

File: docs/blog/v1.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+
+### remark-mdx
+
+`remark-mdx` is the syntactic extension for MDX in remark.  It provides the parsing functionality for
+MDX as a _remark plugin_.  It’s a bit meta.  With `remark-mdx` you can parse MDX documents
+and perform transformations before writing them back to MDX strings.  This allows tools like prettier
+to use the exact same parser as MDX core by installing the library.
```
Comment (wooorm) [inline docs/blog/v1.md]: Also an attempt at a rewrite, not necessarily better!

```text
`remark-mdx` is the syntactic extension for MDX in remark.  It provides the parsing functionality for
MDX as a _[remark](#) plugin_.  That sounds a bit meta.  What it means is that before we had the syntax parsing bits _in_ the library (unusable from the outside), and now it’s externalised (usable from the outside).  This is useful if you want to inspect or transform MDX documents.  For example, it allows tools like prettier to use the exact same parser as us by using the new plugin to format MDX documents.  Or you use `remark-mdx` in combination with [remark-lint](#) to check your MDX.
```

File: docs/blog/v1.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+
+### Custom pragma
+
+With v1 we’ve moved away from using `MDXTag` and are using a custom pragma which wraps `React.createElement`.
```
Comment (wooorm) [inline docs/blog/v1.md]: Should we have an example here? before, after? showing that it’s more legible?
Another random musing: Is there a way we could not name drop React, but keep it vague, so naming Vue later has more effect?

File: docs/blog/v1.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+We decided to update this approach so the JSX output is more legible, lighterweight, and more customizable.
+**This means that we can add support for any library that speaks JSX!**
```
Comment (wooorm) [inline docs/blog/v1.md]: Is `library that speaks JSX` the correct jargon? I’m not sure what the term library expands to here, React, Vue, etc? The term Library is also previously used as `MDX library` relating to our core v. the format, so I’d try and use a different term here. Should it be framework? Pragma?

File: docs/blog/v1.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+
+Special thanks to [@christopherbiscardi](https://christopherbiscardi.com) for implementing this feature.
```
Comment (wooorm) [inline docs/blog/v1.md]: Maybe link to GH / Twitter? I’d expect @mentions to go to a social profile.

File: docs/blog/v1.md [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+[Read the blog post](/blog/custom-pragma)
+
+### 📚 Documentation
+
+**Good libraries need great docs**, so we’ve been working on adding content and improving the overall documentation
+website experience.
+
+*   [Search (thanks Algolia)](https://mobile.twitter.com/4lpine/status/1114270174096412672)
+*   [Guides](https://mdxjs.com)
+*   [Automatic deployment via ZEIT](https://zeit.co)
```
Comment (wooorm) [inline docs/blog/v1.md]: Should this link be more specific?

File: docs/blog/v1.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+*   [Custom Gatsby theme](https://gatsbyjs.org)
```
Comment (wooorm) [inline docs/blog/v1.md]: Should this link be more specific?

File: docs/blog/v1.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+*   Reorganization and misc updates
```
Comment (wooorm) [inline docs/blog/v1.md]: `misc updates` sounds small. To me it feels like the docs are completely rewritten, and I’d suggest someone who previously read the docs to re-read them again, so I would drop this line and mention rewriting in the docs intro paragraph.

File: docs/blog/v1.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+
+Special thanks to [@jxnblk](https://jxnblk.com) and [@wooorm](https://wooorm.com) for their help improving the
+docs and updating the build tooling.
+
+## Breaking changes
+
+Unfortunately, we’ve had to introduce a few breaking changes, but we’ve written a migration guide which
+will walk you through the steps needed to upgrade.
```
Comment (wooorm) [inline docs/blog/v1.md]: This may be very west coast of you, compared to my dutch rudeness, but I wouldn’t start with `Unfortunately,`.
MDX was LTE 1, so I feel like people saw breaking changes coming.
I’d start with something about being significantly better, real-world testing resulting in much better ideas compared to a year ago, and then *even having a migration guide*!

File: docs/blog/v1.md [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+
+*   🚨`@mdx-js/tag` is replaced by `@mdx-js/react` and an `@mdx` pragma - [migration guide](/migrating/v1#pragma)
+*   MDXProvider now merges component contexts when nested - [migration guide](/migrating/v1#mdxprovider)
+*   React support now requires `>= 16.8` in `@mdx-js/react` - [migration guide](/migrating/v1#react)
+
+[Read the full v1 migration guide](/migrating/v1)
+
+## Deprecations
+
+Luckily, we only need to introduce one deprecation.  The `mdPlugins` and `hastPlugins` options have been renamed
```
Comment (wooorm) [inline docs/blog/v1.md]: This section doesn’t need so much focus, I think it could be a bullet point in the previous section!

File: docs/blog/v1.md [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+*   [Docz](https://docz.site)
+*   [MDX Deck](https://github.com/jxnblk/mdx-deck)
+*   [Next.js](https://nextjs.org)
+*   [Gatsby](https://gatsbyjs.org)
+*   [AST Explorer](https://astexplorer.net)
+*   [Vue support (alpha)](/vue)
+*   [Demoboard](https://frontarm.com/demoboard/)
+*   [Editors](/editors)
+
+### Numbers
```
Comment (wooorm) [inline docs/blog/v1.md]: This section should be before ecosystem, start with the bang!

File: docs/blog/v1.md [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+
+*   [Prettier](https://prettier.io)
+*   [Docz](https://docz.site)
+*   [MDX Deck](https://github.com/jxnblk/mdx-deck)
+*   [Next.js](https://nextjs.org)
+*   [Gatsby](https://gatsbyjs.org)
+*   [AST Explorer](https://astexplorer.net)
+*   [Vue support (alpha)](/vue)
+*   [Demoboard](https://frontarm.com/demoboard/)
+*   [Editors](/editors)
```
Comment (wooorm) [inline docs/blog/v1.md]: The above list is a bit of a mix, they’re cool things, but what makes them be in this list? Vue support is very different from prettier!

File: docs/blog/v1.md [hunk 14]
```diff
@@ -0,0 +1,188 @@
...
+*   MDXs: [#454](https://github.com/mdx-js/mdx/issues/454)
+*   Interleaving: [#195](https://github.com/mdx-js/mdx/issues/195)
+*   Global shortcodes: [#508](https://github.com/mdx-js/mdx/pull/508)
+*   Stable Vue support: [#238](https://github.com/mdx-js/mdx/issues/238)
+*   Blocks: MDX WYSIWYG: [blocks/blocks](https://github.com/blocks/blocks)
+*   MDX playground inspired by AST Explorer: [#220](https://github.com/mdx-js/mdx/issues/220)
+*   New splash page: [#444](https://github.com/mdx-js/mdx/issues/444)
+*   Showcase page: [#414](https://github.com/mdx-js/mdx/issues/414)
+
+### Vue support
```
Comment (wooorm) [inline docs/blog/v1.md]: This is much cooler than being down here! It’s usable, right? So add it somewhere at the top! And if there are caveats, describe them here and link to it from the top.

File: docs/blog/v1.md [hunk 15]
```diff
@@ -0,0 +1,188 @@
...
+
+Supporting Vue is an important goal for MDX and is one of the primary reasons we’ve rearchitected MDX to use
+custom pragma.  We’re delighted that **we now have an alpha version for Vue working**.  If anyone from the Vue
+community wants to give it a try and provide feedback we’d be eternally grateful.
```
Comment (wooorm) [inline docs/blog/v1.md]:
```
If anyone from the Vue community wants to give it a try and provide feedback we’d be eternally grateful.
```

Too soft, can be bolder, ask for help! `We’d love it if the Vue community...`

File: docs/blog/v1.md (hunk 13 above)
Comment (johno) [reply docs/blog/v1.md]: Yeah, I kinda arbitrarily added some seemingly relevant things that are new or built around the format 🤷‍♂️

File: docs/blog/v1.md (hunk 14 above)
Comment (johno) [reply docs/blog/v1.md]: My thought on keeping it lower is that it's only an alpha and will have it's own release announcement when more thoroughly tested.
